src/parser/monk/mistweaver/modules/spells/EnvelopingMists.js

```diff
@@ -45,14 +61,15 @@ class EnvelopingMists extends Analyzer {
       return;
     }
 
-    if ((spellId === SPELLS.GUSTS_OF_MISTS.id) && (this.lastCastTarget === event.targetID)) {
+    if ((spellId === SPELLS.GUSTS_OF_MISTS.id) && (this.lastCastTarget === event.targetID) && this.numberToCount >0) {
       this.gustProc += 1;
       this.gustsHealing += (event.amount || 0) + (event.absorbed || 0);
+      this.numberToCount--;
     }
 
     if (this.combatants.players[targetId]) {
       if (this.combatants.players[targetId].hasBuff(SPELLS.ENVELOPING_MIST.id, event.timestamp, 0, 0) === true) {
-        this.healingIncrease += calculateEffectiveHealing(event, EVM_HEALING_INCREASE);
+        this.healingIncrease += calculateEffectiveHealing(event, this.EVM_HEALING_INCREASE);
```

Comment:
Same stylistic item as above. Just use camel case for variables, and full capitalization for constants. This should be `evmHealingIncrease`

Reply:
I was just reusing the old variable name but I'll change it